src/main/java/com/rsmart/kuali/kfs/cr/batch/CheckReconciliationImportStep.java

```diff
@@ -128,37 +154,66 @@ public class CheckReconciliationImportStep extends CuAbstractStep {
      */
     public boolean execute(String jobName, Date jobRunDate) throws InterruptedException {
         LOG.info("Started CheckReconciliationImportStep @ " + (new Date()).toString());
-        
+
+        List<Pair<String, String>> prefixMappings = getParameterPrefixMappings();
```

Comment:
Asking: Do we need to ensure that the file name prefixes are not configured for more than one parameter prefix value?  I can see where the same file name prefixes could use the same system parameters for their processing  but from a functionality standpoint does the opposite make sense and do we need to check for it?

Example that could legitimately be valid:
BankA and BankB both send files in the same format where the same system parameters are used so CR_PARAMETER_PREFIX_MAPPINGS has a value of "BankA_file_prefix_=;BankB_file_prefix_="

But is this valid?:
Same bank file format is setup for two different parameter prefixes so CR_PARAMETER_PREFIX_MAPPINGS has a value of "BankA_file_prefix_=Param_prefixY_;BankA_file_prefix_=Param_prefixZ_"

Reply:
It's fine for two file prefixes to share the same parameter prefix if necessary. However, if a file prefix appears in the mappings more than once, then only the first related mapping will actually have an effect, since the corresponding data files are moved/archived after the first mapping gets processed.

I don't think it's necessary to perform a check for duplicates, but I'll see if I can update the parameter description to discourage the use of duplicates.

Reply:
Actually, upon further consideration, I think it makes sense to add some duplicate filename prefix checking as part of this work. I'll commit the changes once I've finished some quick local validation.